PeerioTechnologies/peerio-mobile
# feat: remove purpose field in room creation [branch ch12374]

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/12374/mobile-remove-purpose-field-from-room-creation
#### Testing instructions  
Create a room successfully with the absence of the "purpose" field. After room has been created, check that the purpose field (in channel info) is empty. 

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
